asset/js/ets_woo_qa_script.js

```diff
@@ -38,29 +38,38 @@ jQuery(document).ready(function(){
 		jQuery(".ets-dis-message-error").text("");
 	});
 
-	jQuery('#ets-load-more').click(function(e){
-		e.preventDefault();  
-		let submit = jQuery ("#ets-qus-form").serialize(); 
-		let qalength = jQuery('.ets_pro_qa_length p').text(); 
-		let offset = jQuery('#ets_product_qa_length p').text(); 
-		if ( typeof offset == 'undefined' ) 
-			offset = 0 ; 
-		jQuery.ajax({ 
-			url: etsWooQaParams.admin_ajax,
-			type: 'GET',  
-			dataType: "JSON",
-			data: 'action=ets_product_qa_load_more&' + submit + '&offset=' + offset +'&load_qa_nonce=' + etsWooQaParams.load_qa_nonce,
-			success: function(res) {  
-				offset = res.offset;  
-				jQuery('.table1').append(res.htmlData);  
-				jQuery('.ets-accordion-response-add').append(res.htmlData);
-				jQuery('#ets_product_qa_length p').html(offset).hide();
-				if(offset >= qalength ){
-					jQuery("#ets-load-more").hide();
-				} 
-            }
-        }); 
-	}); 
+	function handleLoadMoreClick(buttonId, tableId, accordionId,QaLength,QaOffset) {
+	    jQuery(buttonId).click(function(e) {
+	        e.preventDefault();
+	        let submit = jQuery("#ets-qus-form").serialize(); 
+	        let qalength = jQuery(QaLength).find('p').text();
+	        let offset = jQuery(QaOffset).find('p').text(); 
+	        if (typeof offset == 'undefined') 
+	            offset = 0; 
+	        jQuery.ajax({ 
+	            url: etsWooQaParams.admin_ajax,
+	            type: 'GET',  
+	            dataType: "JSON",
+	            data: 'action=ets_product_qa_load_more&' + submit + '&offset=' + offset +'&load_qa_nonce=' + etsWooQaParams.load_qa_nonce,
+	            success: function(res) {
+	             
+	                offset = res.offset;  
+	                jQuery(tableId).append(res.htmlData);  
+	                jQuery(accordionId).append(res.htmlData);
+	                jQuery(QaOffset).find('p').html(offset).hide();
+	                if(offset >= qalength ){
+	              
+	                    jQuery(buttonId).hide();
+	                } 
+	            }
+	        }); 
+	    });
+	}
+
+	handleLoadMoreClick('#ets-load-more-1', '.ets-list-table-1', '.ets-accordion-list-qa-1','.ets_pro_qa_length_1','#ets_product_qa_length_1');
+
+	handleLoadMoreClick('#ets-load-more-2', '.ets-list-table-2', '.ets-accordion-list-qa-2','.ets_pro_qa_length_2','#ets_product_qa_length_2');
```

Comment:
Why `#ets-load-more-2`  2nd ID was added?

Reply:
It is used when we put the shortcode and default functionality together when we don't use this it dismisses the shortcode load more.

Reply:
when we have to click first load more default loads more when we do not separate it also load second.